core/src/main/java/org/commonjava/maven/ext/core/impl/FinalGroovyManipulator.java

```diff
@@ -50,15 +51,15 @@ public FinalGroovyManipulator(ModelIO modelIO, FileIO fileIO, PomIO pomIO)
     }
     /**
      * Initialize the {@link GroovyState} state holder in the {@link ManipulationSession}. This state holder detects
-     * version-change configuration from the Maven user properties (-D properties from the CLI) and makes it available for
-     * later.
+     * version-change configuration from the Maven user properties (-D properties from the CLI) and makes it available
+     * for later.
      */
     @Override
     public void init( final ManipulationSession session ) throws ManipulationException
     {
-        GroovyState gs = new GroovyState( session.getUserProperties() );
+        final State state = new GroovyState( session.getUserProperties() );
         this.session = session;
-        session.setState( gs );
+        session.setState( state );
```

Comment:
Any particular reasons for the renaming ... :-)

Reply:
No, but I figured that `setState` makes more sense to take ` `state` parameter. I can change it back if you want.

Reply:
No its fine, I was just curious!